public/js/teacher.js

```diff
@@ -0,0 +1,13 @@
+(function() {
```

Comment:
Currently this file is identical to "student.js". Are both needed? Will code be added in the future that will differentiate them? If not, can we just combine these into one file?